internal/clientconn/conninfo/conn_info_test.go

```diff
@@ -25,26 +26,30 @@ func TestGet(t *testing.T) {
 	t.Parallel()
 
 	for name, tc := range map[string]struct {
-		peerAddr string
+		peer  netip.AddrPort
+		local bool
 	}{
-		"EmptyPeerAddr": {
-			peerAddr: "",
+		"Unix": {
+			local: true,
 		},
-		"NonEmptyPeerAddr": {
-			peerAddr: "127.0.0.8:1234",
+		"Local": {
+			peer:  netip.MustParseAddrPort("127.42.7.1:1234"),
+			local: true,
```

Comment:
Shall we add an IPv6?

Reply:
I'm surprised not only Unix socket, 127.0.0.1, and :111 are acceptable as 'localhost exception'.

Reply:
You mean just add test for IPv6? The implementation should just work for either IPv4 or IPv6. What value would you like to test?